togglemenu.js

```diff
@@ -0,0 +1,19 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+
+openNav();
+closeNav();
+
+const navLinks = document.querySelectorAll('.overlay-item');
+const navMenu = document.getElementById('.overlay-content');
+
+function linkshandler() {
+navMenu.classList.remove('show');
+}
```

Comment:
- [OPTIONAL ] Great job working on the menu so far but I think in small screen sizes like 336px it would be great if you fix the menu according to it so not let the first section appear in these small sizes but great job on the noraml mobile view it is fine ✅📌💯🔥♻✨✨